modules/common/src/main/java/org/opencastproject/mediapackage/AdaptivePlaylist.java

```diff
@@ -330,15 +320,12 @@ static void hlsRewriteFileReference(File srcFile, Map<String, String> mapNames)
    *           if failed
    */
   static void hlsRewriteFileReference(File srcFile, File destFile, Map<String, String> mapNames) throws IOException {
-    FileWriter hlsReWriter = null;
-    BufferedReader br = null;
     // Many tags reference URIs - not all are dealt with in this code, eg:
     // "#EXT-X-MAP:", "#EXT-X-MEDIA:", "#EXT-X-I-FRAME-STREAM-INF:", "#EXT-X-SESSION-DATA:", "#EXT-X-KEY:",
     // "#EXT-X-SESSION-DATA:"
-    try {
-      hlsReWriter = new FileWriter(destFile.getAbsoluteFile(), false);
-      br = new BufferedReader(new FileReader(srcFile));
-      String line = null;
+    try (FileWriter hlsReWriter = new FileWriter(destFile.getAbsoluteFile(), false);
+         BufferedReader br = new BufferedReader(new FileReader(srcFile))) {
```

Comment:
The old implementation could lead to NullPointerExceptions, this one should be safe with less unnecessary code.